### PR TITLE
make OpenAPIConfig works

### PR DESCRIPTION
**Pull request by jeremyxu2010**
I notice wrong golang struct coping causes OpenAPIConfig not working, so I fixed it. see #380

### CODE REVIEW

File: pkg/cmd/server/start.go [hunk 1]
```diff
@@ -313,6 +314,11 @@ func (o *ServerOptions) RunServer(stopCh <-chan struct{}, title, version string,
 	if err != nil {
 		return err
 	}
+
+	aggregatedAPIServerConfig.RecommendedConfig.Config.OpenAPIConfig = genericapiserver.DefaultOpenAPIConfig(GetOpenApiDefinition, openapinamer.NewDefinitionNamer(builders.Scheme))
+	aggregatedAPIServerConfig.RecommendedConfig.Config.OpenAPIConfig.Info.Title = title
+	aggregatedAPIServerConfig.RecommendedConfig.Config.OpenAPIConfig.Info.Version = version
+
 	genericConfig := aggregatedAPIServerConfig.RecommendedConfig.Config
```
Comment (yue9944882) [inline pkg/cmd/server/start.go]:
```suggestion
      genericConfig := &aggregatedAPIServerConfig.RecommendedConfig.Config
```

thank you for fixing this. how you think about another pass? @jeremyxu2010

Comment (jeremyxu2010) [reply pkg/cmd/server/start.go]: done as you suggested @yue9944882

**Pull request merged**